docs/feature_led_indicators.md

```diff
@@ -11,6 +11,7 @@ QMK provides methods to read 5 of the LEDs defined in the HID spec:
 * Kana
 
 There are three ways to get the lock LED state:
+* Configuration options in `info.json` 
```

Comment:
This is not how you "get" the lock LED state programmatically, it's how you define the pins used to control the LEDs.